contrib/tetragon-rthooks/cmd/setup/patch-crio-conf.go

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package main
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	crioconf "github.com/cri-o/cri-o/pkg/config"
+	"github.com/pelletier/go-toml"
+	tomlparser "github.com/pelletier/go-toml/v2/unstable"
+)
+
+// copy this from crio since it does not export it
+type tomlConfig struct {
+	Crio struct {
+		crioconf.RootConfig
+		API     struct{ crioconf.APIConfig }     `toml:"api"`
+		Runtime struct{ crioconf.RuntimeConfig } `toml:"runtime"`
+		Image   struct{ crioconf.ImageConfig }   `toml:"image"`
+		Network struct{ crioconf.NetworkConfig } `toml:"network"`
+		Metrics struct{ crioconf.MetricsConfig } `toml:"metrics"`
+		Tracing struct{ crioconf.TracingConfig } `toml:"tracing"`
+		Stats   struct{ crioconf.StatsConfig }   `toml:"stats"`
+		//NRI     struct{ *nri.Config }            `toml:"nri"`
```

Comment:
Do we need this line?

Reply:
removed the whole structure, it wasn't been used.